docs/ci-integration.md

```diff
@@ -1,3 +1,9 @@
+# ⚠ ignore everything here
```

Comment:
figured id leave it for the sake of minimizing upstream conflicts but i think its unlikely that documentation will ever change because they dont seem to care about ci